bundles/AdminBundle/Resources/views/Admin/Misc/iconList.html.twig

```diff
@@ -126,7 +129,15 @@
     {% endfor %}
 </table>
 
+{{ pimcore_minimize_scripts(['pimcore/common.js', 'pimcore/functions.js','pimcore/helpers.js']) | raw }}
```

Comment:
```suggestion
<script src="/bundles/pimcoreadmin/js/pimcore/common.js"></script>
<script src="/bundles/pimcoreadmin/js/pimcore/functions.js"></script>
<script src="/bundles/pimcoreadmin/js/pimcore/helpers.js"></script>
```
We want to get rid of `pimcore_minimize_scripts`, so let's just use the script directly - no real benefit in this case anyway ☺️